src/http/api/routes/files.js

```diff
@@ -2,7 +2,66 @@
 
 const resources = require('./../resources')
 const mfs = require('ipfs-mfs/http')
+const fs = require('fs')
+const path = require('path')
+const tempy = require('tempy')
+const multipart = require('ipfs-multipart')
+const pull = require('pull-stream')
+const toPull = require('stream-to-pull-stream')
+const pushable = require('pull-pushable')
 
+const streams = []
+const filesDir = tempy.directory()
```

Comment:
Do we have a plan for what happens with temp files produced by failed/aborted  uploads?
Leaving them forever may be a surface for self-administered DoS attacks :)